shared/teams/confirm-modals/delete-channel/index.tsx

```diff
@@ -18,23 +20,26 @@ const Header = () => (
   </>
 )
 
-const getTeamSelectedCount = (state: Container.TypedState, teamID: TeamsTypes.TeamID) => {
+const getTeamSelectedCount = (state: Container.TypedState, teamID: Types.TeamID) => {
   return state.teams.teamSelectedChannels.get(teamID)
 }
 
 const DeleteChannel = (props: Props) => {
-  const teamID = Container.getRouteProps(props, 'teamID', TeamsTypes.noTeamID)
-  const channelIDs = Container.useSelector(state => getTeamSelectedCount(state, teamID))
+  const teamID = Container.getRouteProps(props, 'teamID', Types.noTeamID)
+  const routePropChannel = Container.getRouteProps(props, 'conversationIDKey', undefined)
+  const channelIDs = routePropChannel
+    ? [routePropChannel]
+    : Container.useSelector(state => getTeamSelectedCount(state, teamID))
```

Comment:
this isn't allowed. you can't conditionally use a hook like this

Reply:
whoops